snmp/datadog_checks/snmp/data/profiles/cisco-3850.yaml

```diff
@@ -1,3 +1,4 @@
+# Backward compatibility shim. Prefer the Cisco Catalyst profile directly
```

Comment:
Given that sysobjectid is not copied over when extending from a profile, we could in principle really reduce this compat profile to:

```yaml
extends:
  - cisco-catalyst.yaml

sysobjectid: 1.3.6.1.4.1.9.1.1745
```